docs/developer-guide/nodejs-api.md

```diff
@@ -56,8 +56,8 @@ const { ESLint } = require("eslint");
   const results = await eslint.lintFiles(["lib/**/*.js"]);
 
   // 3. Format the results.
-  const formatter = await eslint.loadFormatter("stylish");
-  const resultText = formatter.format(results);
+  const loadedFormatter = await eslint.loadFormatter("stylish");
```

Comment:
I’m not sure this change makes sense. The variable name doesn’t need to match the type name.